app/models/report.rb

```diff
@@ -21,8 +21,8 @@ def created_on
     created_at.to_date
   end
 
-  def report_ids_in_content
-    content.scan(/#{DOMAIN}[0-9]+/)
+  def find_report_url
```

Comment:
```suggestion
  def find_report_urls
```

配列が返るので複数形の方が良さそうです。